cumulusci/cli/cci.py

```diff
@@ -833,7 +836,10 @@ def org_default(config, org_name, unset):
         config.keychain.unset_default_org()
         click.echo(f"{org_name} is no longer the default org.  No default org set.")
```

Comment:
🤔 :
```suggestion
        click.echo(f"Unsetting default org...")
        click.echo(f"No default org set.")
```

Reply:
@jstvz Should I do this now or should it happen in a different PR? Happy to commit the suggestion now if that's ok.